Scientific_workflows/FractionalCoverOfWaterProject/Manual_classification_workflow.ipynb

```diff
@@ -0,0 +1,994 @@
+{
```

Comment:
<p>Can you avoid the use of globals here? Pass xr_results as an argument</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/663/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Yes, I can, but I'd rather not, for now. To make it neater will require much more time in editing the code. Besides, it requires more work to debug everything within the bokeh function. IF anything is wrong, the plot simply doesn't show or the interactivity doesn't work as it's supposed to. At the time, only a browser console output can pin point the problem. Not being proficient in javascript, I'd like to avoid the pain. </p><p> </p><p>I wouldn't say the plot shows a "correct" or "decent", let alone "universal",  approach to do bokeh interactive plotting. It's a "working" way by trial and a reference for repo in the future. Hence I will keep it as what it is until I gain better understanding of how bokeh works.</p>